### PR TITLE
[Web][SIMS #906] Show awards based on offering intensity

### PR DESCRIPTION
**Pull request by dheepak-aot**
- [x] Based on the Awards located https://github.com/bcgov/SIMS/wiki/Acronyms-&-Terminology#awards display only FT awards when it's a FT offering and PT awards when it's a PT offering

### CODE REVIEW

**Review by andrewsignori-aot — APPROVED** [review]
Looks good 👍

**Review by guru-aot — APPROVED** [review]
LGTM

File: sources/packages/web/src/components/common/AwardTable.vue [hunk 1]
```diff
@@ -0,0 +1,135 @@
+<template>
+  <v-table class="bordered">
+    <thead>
+      <tr>
+        <th scope="col" class="text-left">Loan/grant type</th>
```
Comment (ann-aot) [inline sources/packages/web/src/components/common/AwardTable.vue:5]: why `scope="col"`?

Comment (dheepak-aot) [reply sources/packages/web/src/components/common/AwardTable.vue:5]: It is from sonar rule to use th always scope or id.

https://rules.sonarsource.com/html/RSPEC-1102

**Review by ann-aot — APPROVED** [review]
Good Work @dheepak-aot  👍

**Pull request merged**